Assets/Scripts/ConnectionManagement/ConnectionState/StartingHostState.cs

```diff
@@ -1,9 +1,13 @@
 using System;
+using ConnectionManagement.ConnectionState;
 using Unity.BossRoom.Infrastructure;
 using Unity.BossRoom.UnityServices.Lobbies;
 using Unity.Multiplayer.Samples.BossRoom;
 using Unity.Netcode;
 using Unity.Netcode.Transports.UTP;
+using Unity.Networking.Transport.Relay;
+using Unity.Services.Relay;
+using Unity.Services.Relay.Models;
```

Comment:
Same here, those three new usings are not necessary and UTP can be removed now as well